src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -176,7 +176,7 @@ public boolean isRequired()
                         return new Command.Choice(entry.getKey(), entry.getValue().toString());
                     else if (entry.getValue() instanceof Double)
                         return new Command.Choice(entry.getKey(), (double) entry.getValue());
-                    return new Command.Choice(entry.getKey(), (long) entry.getValue());
+                    return new Command.Choice(entry.getKey(), ((Number) entry.getValue()).longValue());
```

Comment:
Did you test that the cast for double above this works?

Reply:
Can't even test because apparently "Release version 8 not supported" lol

Reply:
I changed it to `((Number) entry.getValue()).doubleValue()` just in case.